openadapt/minigpt4_demo_modal.py

```diff
@@ -0,0 +1,244 @@
+# import argparse
```

Comment:
LGTM (haven't tested it, but makes sense)

Reply:
We decided to pause work on this since there were some runtime issues and the `transformers.agent` seems like a better API.

@angelala3252 can you please comment on what's left to get this merged?